turbopack/crates/turbo-tasks-backend/src/kv_backing_storage.rs

```diff
@@ -445,6 +467,15 @@ impl<T: KeyValueDatabase + Send + Sync + 'static> BackingStorage
             .unwrap_or_default()
     }
 
+    fn invalidate(&self) -> Result<()> {
+        if let Some(base_path) = &self.base_path {
+            // `base_path` can be `None` for a `NoopKvDb`
+            invalidate_db(base_path)?;
+            self.database.prevent_writes()
```

Comment:
nit: prevent writes first?  i find myself wondering if `invalidate_db` might partially fail.  e.g. create the file but fail to write the payload.